ProjFS.Mac/PrjFSKextTests/HandleOperationTests.mm

```diff
@@ -853,5 +1185,21 @@ - (void) testWriteDataToForkedPath {
     XCTAssertTrue(MockCalls::CallCount(ProviderMessaging_TrySendRequestAndWaitForResponse) == 0);
 }
 
+- (void) testIneligibleFilesystemType
```

Comment:
Were we just missing coverage for this case before?

Reply:
Commit c8b4573 moves the eligibility check outside the `ShouldHandle…` function, so there's a new code path in `HandleVnodeOperation`, and this path isn't covered by existing tests, no.